src/webaudio.js

```diff
@@ -298,19 +302,23 @@ export default class WebAudio extends util.Observer {
              * output. Here we create an HTMLAudioElement, connect the
              * webaudio stream to that element and setSinkId there.
              */
-            let audio = new window.Audio();
-            if (!audio.setSinkId) {
+            if (!this.sinkAudioElement) {
+                this.sinkAudioElement = new window.Audio();
+                this.sinkAudioElement.autoplay = true;
+            }
+            if (!this.sinkAudioElement.setSinkId) {
                 return Promise.reject(
                     new Error('setSinkId is not supported in your browser')
                 );
             }
-            audio.autoplay = true;
```

Comment:
> is this autoplay option necessary? if so, add a comment why
It was already here, I do not think it's necessary since we're not invoking playback via `.play()`.

Reply:
ah yea missed that. I guess it doesn't hurt to leave it in for now.

Reply:
@thijstriemstra Interesting. I tested with and without the autoplay. It no longer allocates the items in media-internals but doesn't playback any audio to the chosen sink. 
So now, I think a comment about this being load bearing is important.

Reply:
if you want to add another comment, sure, please do

Reply:
also, please make sure to test with firefox as well.

Reply:
> also, please make sure to test with firefox as well.

I'm having trouble getting setSinkId/enumerateDevices to work in Firefox. and a harder time following the tracker. 
https://bugzilla.mozilla.org/show_bug.cgi?id=1610319
https://bugzilla.mozilla.org/show_bug.cgi?id=1493982

Reply:
you essentially didn't change the implementation, only the cleanup, and I tested it in Firefox (first) before so I can merge this and if you see any issues with Firefox, please open a ticket or new PR?